src/current/_includes/releases/v22.2/v22.2.13.md

```diff
@@ -0,0 +1,138 @@
+- Previously, cross-database type references were allowed through [`CREATE TABLE..AS`](../v22.2/create-table-as.html) statements if the source table was from another database and any of its columns was of a [user-defined type](../v22.2/create-type.html). This introduced a bug where the source table could be dropped and the type could not be found for the `CREATE TABLE..AS` table. This fix disallows such `CREATE TABLE..AS` statements. [#105619][#105619]
+- Fixed a bug that could cause an upgrade from v22.1 to v22.2 to take a long time, and sometimes not finish if there were a lot of tables that were being created and dropped regularly. [#99936][#99936]
+- Fixed a bug in the [geospatial](../v22.2/spatial-indexes.html#how-cockroachdbs-spatial-indexing-works) cartesian bounding box type that had incorrect behavior when comparing boxes with `NaN` values. [#105829][#105829]
+- Fixed a bug that manifested itself in error messages containing `failed to drop all of the relevant elements` when executing DDL statements with the [declarative schema changer](../v22.2/online-schema-changes.html#declarative-schema-changer). This error message actually meant that a concurrent schema change is ongoing. Instead, this now behaves as expected and waits for it to finish. [#106300][#106300]
+- Fixed a bug where the SQL activity page was not loading data during v22.1 to v22.2 ugprades. [#105664][#105664]
+- Previously, flushing SQL stats that resulting in an update statement in mixed version v22.1 and v22.2 clusters would fail due to attempting to update the `index_recommendations` column, which is new in v22.2. [#105664][#105664]
+- Fixed a panic that occurred when executing `CREATE AS` statements with a [`pg_catalog.pg_prepared_statements`](../v22.2/pg-catalog.html) or `pg_catalog.pg_cursors` source. For example: `CREATE TABLE t AS SELECT * FROM pg_catalog.pg_prepared_statements;` or `CREATE MATERIALIZED VIEW v AS SELECT * FROM pg_catalog.pg_prepared_statements;`. [#106288][#106288]
+- Fixed a bug where some [secondary indexes](../v22.2/schema-design-indexes.html) would incorrectly be treated internally as primary indexes, which could cause some schema change operations to fail. The bug could occur if [`ALTER PRIMARY KEY`](../v22.2/alter-table.html#alter-primary-key) was used on v21.1 or earlier, and the cluster was upgraded. [#106428][#106428]
+- CockroachDB will now return an error during [user-defined function](../v22.2/user-defined-functions.html) creation if an input argument has type `RECORD`. [#105734][#105734]
+- [`SHOW SCHEMAS FROM db_name`](../v22.2/show-schemas.html) will no longer incorrectly show schemas from the current database when the current database has a schema named `db_name`. It will now look up the database name instead of the previous schema look up logic. [#106198][#106198]
```

Comment:
```suggestion
- [`SHOW SCHEMAS FROM db_name`](../v22.2/show-schemas.html) will no longer incorrectly show schemas from the current database when the current database has a schema named `db_name`. It will now look up the database name instead of using the schema look up logic. [#106198][#106198]
```